kolibri/plugins/learn/assets/src/views/ChannelCard.vue

```diff
@@ -113,6 +117,11 @@
         required: false,
         default: null,
       },
+      isPinned: {
```

Comment:
I think this prop is misleading. I think the wifi icon is meant to specify whether the resource is available or not. @jtamiace , @marcellamaki ?